worker.js

```diff
@@ -63,40 +85,51 @@ var getBucketList = function (cb) {
 var readBuckets = function (tasks) {
   console.info('--- Started indexing all buckets ---');
   async.parallel(tasks,
```

Comment:
What should the limit be? There are only 2 buckets right now on OIN. We can move this discussion to #87 once this PR for the local indexing is merged in.